docs/reference/release-notes/highlights-7.2.0.asciidoc

```diff
@@ -10,5 +10,14 @@ coming[7.2.0]
 //Installation and Upgrade Guide
 
 // tag::notable-highlights[]
+[discrete]
+==== {dataframes-cap}
+
+beta[] You can now transform your data with
+{stack-ov}/ml-dataframes.html[data frames]. There is a new {kib} wizard that
+guides you through the process of creating a {dataframe-transform} job to pivot
```

Comment:
change to .. `... creating a {dataframe-transform} to pivot and summarize your data and store it in a new index.`

`job` is not needed .. `summarize` allows us to further reinforce that we are aggregating.

Reply:
Thanks, I like it!  I've implemented that change.